webapp/tests/forms/test_validators.py

```diff
@@ -52,6 +52,39 @@ def test_if_not_only_decimal_raise_value_error(self):
                 validator.__call__(self.form, self.field)
 
 
+class TestMaximumLength:
+    @pytest.fixture()
+    def field(self):
+        return SteuerlotseStringField()
+
+    @pytest.fixture()
+    def form(self):
+        return SteuerlotseBaseForm()
+
+    def test_if_string_shorter_than_max_then_return_no_validation_error(self, field, form):
+        validator = MaximumLength(5)
+        field.data = "1337"
+        try:
+            validator(form, field)
+        except ValidationError:
```

Comment:
If we don't expect a error, we don't have to catch one. If a validation error is thrown, the test will turn red anyway.